### DISCUSSION TITLE
Efficient Amortised Bayesian Inference for Hierarchical and Nonlinear Dynamical Systems

### DISCUSSION THREAD

**Discussion by MichaelClerx** [pints-team/pints#1613] [Ideas/Science]
@mirams posted https://arxiv.org/abs/1905.12090

Someone should have a quick look at this

**Comment by martinjrobins** [comment]
I like this one. @ben18785, this is along the lines of what I was thinking about variational autoencoders,  they look like a nice way to incorporate neural networks into inference

(Disclaimer: "along the lines of what I was thinking" means I had vague thoughts that variational autoencoders could be used to learn a posterior, definitely not saying I understand them in any meaningful way!)

**Comment by ben18785** [comment]
It looks cool but I must admit I don't really understand it. Martin, next time I'm about I'm going to quiz you about it!

**Comment by martinjrobins** [comment]
Sounds good, think I'm slowly getting an understanding. The nice thing about this approach is that it gives you flexibility to define your model as an ODE, Neural Network, GP, or whatever (or a combination of a few of those). Only requirement is that it is differentiable.
